From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Danila Tikhonov <danila@jiaxyga.com>
Date: Fri, 3 Jan 2025 23:52:01 +0300
Subject: [PATCH] soc: qcom: pd_mapper: Add SM8475 compatible

Add the Qualcomm SM8475 platform compatibility leading to SM8350.

Signed-off-by: Danila Tikhonov <danila@jiaxyga.com>
(cherry picked from commit 8b94a1f16bd4fcf77d9ce70e94b74898415455d9)
---
 drivers/soc/qcom/qcom_pd_mapper.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/qcom/qcom_pd_mapper.c b/drivers/soc/qcom/qcom_pd_mapper.c
index 0dc1a79460..c455d70761 100644
--- a/drivers/soc/qcom/qcom_pd_mapper.c
+++ b/drivers/soc/qcom/qcom_pd_mapper.c
@@ -622,6 +622,7 @@ static const struct of_device_id qcom_pdm_domains[] __maybe_unused = {
 	{ .compatible = "qcom,sm8250", .data = sm8250_domains, },
 	{ .compatible = "qcom,sm8350", .data = sm8350_domains, },
 	{ .compatible = "qcom,sm8450", .data = sm8350_domains, },
+	{ .compatible = "qcom,sm8475", .data = sm8350_domains, },
 	{ .compatible = "qcom,sm8550", .data = sm8550_domains, },
 	{ .compatible = "qcom,sm8650", .data = sm8550_domains, },
 	{ .compatible = "qcom,sm8750", .data = sm8550_domains, },
